### ISSUE TITLE
TypeError: text() got an unexpected keyword argument 'squash_space' 

### ISSUE THREAD

**Issue by mknudsen** [gleitz/howdoi#192]
I installed howdoi via homebrew and got an error:

```
$howdoi help
Traceback (most recent call last):
  File "/usr/local/bin/howdoi", line 11, in <module>
    load_entry_point('howdoi==1.1.13', 'console_scripts', 'howdoi')()
  File "/usr/local/Cellar/howdoi/1.1.13/libexec/lib/python2.7/site-packages/howdoi/howdoi.py", line 311, in command_line_runner
    utf8_result = howdoi(args).encode('utf-8', 'ignore')
  File "/usr/local/Cellar/howdoi/1.1.13/libexec/lib/python2.7/site-packages/howdoi/howdoi.py", line 263, in howdoi
    return _get_instructions(args) or 'Sorry, couldn\'t find any help with that topic\n'
  File "/usr/local/Cellar/howdoi/1.1.13/libexec/lib/python2.7/site-packages/howdoi/howdoi.py", line 233, in _get_instructions
    answer = _get_answer(args, question_links)
  File "/usr/local/Cellar/howdoi/1.1.13/libexec/lib/python2.7/site-packages/howdoi/howdoi.py", line 205, in _get_answer
    text = _format_output(_get_text(instructions.eq(0)), args)
  File "/usr/local/Cellar/howdoi/1.1.13/libexec/lib/python2.7/site-packages/howdoi/howdoi.py", line 102, in _get_text
    return element.text(squash_space=False)
```

Manually removing `squash_space=False` from `element.getText(..)` fixed this locally. As I have little understanding of python I have no idea how to fix this without breaking it for people using older versions of PyQuery.

[PyQuery](https://pypi.python.org/pypi/pyquery) website states

> 1.4.0 (2018-01-11)
> Refactoring of .text() to match firefox behavior.

so maybe that's the cause.

**Comment by WindSoilder** [comment]
I think it's because *PyQuery* is not 1.4.0 in your machine, would you please tell me that which version of *PyQuery* is installed on your machine ?

Actually, the fix of #186 makes howdoi don't work with older version of *PyQuery* (which is less than 1.4.0)

**Comment by tra38** [comment]
> I installed howdoi via homebrew

[This line on howdoi.rb](https://github.com/gleitz/howdoi/blob/2a5f6615fada06992107831c2aad31018918e4f8/howdoi.rb#L34) suggests that whenever someone uses home-brew, they install PyQuery 1.2.11 instead of 1.4.0. Could updating that line to the latest version of PyQuery and then reinstalling howdoi using homebrew fix the issue?
